tests/unit/steps/test_steps.py

```diff
@@ -32,21 +32,36 @@
     compare_step_coroutines,
     is_unit_upgrade_step,
 )
+from cou.utils import app_utils
 
 
 async def mock_coro(*args, **kwargs): ...
 
 
+async def _coro(name: str, force: bool = False, units: list | None = None): ...
```

Comment:
Could you add some comments here explaining why the two types of macked coros are define. It would be very confusing to look at from a future traveler's perspective.

Reply:
I will use one coro for all tests, and that will solve this issue.